app/containers/Questions/QuestionsForm.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
Let's reserve suffix 'Form' for components that are actually forms (inputs and submit button). Rename to something else, e.g. QuestionsContainer.